### ISSUE TITLE
Option to change hyperlink and text color sheme

### ISSUE THREAD

**Issue by Oremountainflorian** [ether/etherpad-lite#3960]
Hey everyone! We are really happy about version 1.8.3, the appearance improvements are awesome! I'm pretty new to etherpad and I hope my issue is at the right place here.

## Description

One thing I don't like is the green color sheme for hyperlinks, H1 and some text in the settings and the chat window. Especially in the text field itself this is pretty impractical for many author colors as the hyperlinks are barely readable.

![grafik](https://user-images.githubusercontent.com/44228345/80577261-df5be580-8a06-11ea-97df-d26bbc49eede.png) ![grafik](https://user-images.githubusercontent.com/44228345/80577323-f8649680-8a06-11ea-91a8-790601572177.png)

This also applies if the plain hyperlink/H1 heading is selected (light blue background). 

![grafik](https://user-images.githubusercontent.com/44228345/80577088-93a93c00-8a06-11ea-80d7-28befb3a43d8.png)

## Possible solution

- A darker or not so "neon-like" default hyperlink color could enhance contrast and readability. 
- Furthermore, it would also be nice to change the hyperlink color in case the contrast with the background color was too weak as it is done with normal text (changes from black to white depending on the contrast or with H1 heading). 

## Optional solution

- Provide the option to set the font color sheme (default green) with #skinvariantbuilder or at least provide this option to settings.json. 
- It would also be cool to assign specific color for hyperlinks.

## Question

Basically I would like to set the hyperlink color to deep blue on our etherpad instance. Could you please help me and provide some code how to do that?

**Comment by JohnMcLear** [comment]
@seballot any thoughts on this?  Sounds like a sensible request..

**Comment by JohnMcLear** [comment]
See: https://github.com/ether/etherpad-lite/issues/3641

**Comment by seballot** [comment]
Hi ! Indeed the green url does not fit with author color. I'm always using pads with author_neat that's why I did not noticed that, will try a fix !

**Comment by seballot** [comment]
Ok done, I disabled the coloring of hyperlink when color authorship is on

**Comment by Oremountainflorian** [comment]
@seballot Thanks for your quick fix! Just another question. You said the coloring of hyperlinks is just of when the switch "Authorship colors" in the option is off, right? However, it would be nice also to color the hyperlink when the background color is cleared, even if authorship colors turned on (for better visibility). 

And sorry for my persistence, but I also think the default blue color of hyperlinks like https://github.com/ether/etherpad-lite/issues/3960 is the most intuitive option to color hyperlinks. 

Somehow or other, another sensible feature would be coloring the hyperlink white when it's selected by dragging the mouse. 

![grafik](https://user-images.githubusercontent.com/44228345/80711422-9da76980-8af0-11ea-8279-b4613f6591b1.png)

Best regards and thank you for your commitment, Flo

**Comment by seballot** [comment]
Yes good points. I made the links blue when no authorship color, and also when the authorship is cleared.
Coloring the text when it's selected, not sure it is possible

![link](https://user-images.githubusercontent.com/17404254/80813255-fd257800-8bc9-11ea-85ac-b6c57004ec83.gif)
